main.cpp

```diff
@@ -73,12 +25,33 @@ void testPairToNumber(
     assert(pairNumber == expectedPairNumber);
 }
 
+void printColorCodeManualRef(const std::string &filename) {
```

Comment:
Good implementation to place the manual in a file 👍 
One improvement: This function is placed together with the test code and main() function too. So if I need to use this function, I need to take the test code as well.
Separate this into its own file. In general, it is better to develop new code separately from legacy, whenever possible